c/mtlskt.c

```diff
@@ -23,6 +23,8 @@
 #include "alloc.h"
 #include "utils.h"
 #include "bpxnet.h"
+#include "logging.h"
+#include "zccLogging.h"
```

Comment:
We should not require official messages in a socket library code. In fact, I think we should avoid any zowelog or messages and return proper rc and rsn codes. Remember, this is library code.